merger/fix.go

```diff
@@ -86,10 +86,14 @@ func FixLoads(f *rule.File, knownLoads []rule.LoadInfo) {
 			if id, ok := d.X.(*bzl.Ident); ok {
 				idents = append(idents, id)
 			} else {
-				return
+				return nil
 			}
+		} else if _, ok := x.(*bzl.LoadStmt); ok {
+			return &bzl.StopTraversalError{}
+		} else if id, ok := x.(*bzl.Ident); ok {
+			idents = append(idents, id)
```

Comment:
This could be many idents, e.g. `bar` and `baz` in `foo.bar.baz` or even numeric literals. Could we limit this to the RHS of an assignment expression (or a dot expression in there) for now?

Reply:
Thank you. I've updated the branch and PR description.